src/components/Grid.less

```diff
@@ -0,0 +1,97 @@
+@import (reference) '@cfpb/cfpb-design-system';
+
+// TODO: Do we need to support non-12-column grids?
```

Comment:
Question

Reply:
Pinging @natalia-fitzgerald for the above question